script/release/ci-release-build.js

```diff
@@ -27,6 +27,9 @@ const vstsArmJobs = [
   'electron-woa-testing'
 ]
 
+const circleCIRetryLimit = 10
```

Comment:
teeny nit - for semantic consistency, could we make these `const` variables take more normal capitalized format?
 a la

```js
const APPVEYOR_BUILD_URL = 'https://ci.appveyor.com/api/builds'
const CIRCLECI_PIPELINE_URL = 'https://circleci.com/api/v2/project/gh/electron/electron/pipeline'
const VSTS_URL = 'https://github.visualstudio.com/electron/_apis/build'

const CIRCLECI_RETRY_LIMIT = 10
const CIRCLECI_WAIT_TIME = 10000
```

and place them together at the top?

Reply:
sounds good to me.